File: surf-cloud-core/src/main/java/dev/slne/surf/cloud/core/netty/protocol/packets/cloud/info/CloudServerInfoAction.java
package dev.slne.surf.cloud.core.netty.protocol.packets.cloud.info;

import dev.slne.surf.cloud.api.server.CloudServer;

public enum CloudServerInfoAction {

  /**
   * Updates properties of a {@link CloudServer}
   */
  UPDATE_SERVER_INFO,
  /**
   * Removes a {@link CloudServer}
   */
  REMOVE_SERVER_INFO,
  /**
   * Adds a {@link CloudServer}
   */
  ADD_SERVER_INFO
}

File: surf-cloud-core/src/main/kotlin/dev/slne/surf/cloud/core/netty/protocol/packets/cloud/info/CloudServerInfoAction.kt
package dev.slne.surf.cloud.core.netty.protocol.packets.cloud.info

enum class CloudServerInfoAction {
    /**
     * Updates properties of a [dev.slne.surf.cloud.api.server.CloudServer]
     */
    UPDATE_SERVER_INFO,

    /**
     * Removes a [dev.slne.surf.cloud.api.server.CloudServer]
     */
    REMOVE_SERVER_INFO,

    /**
     * Adds a [dev.slne.surf.cloud.api.server.CloudServer]
     */
    ADD_SERVER_INFO
}
